README.md

```diff
@@ -22,3 +22,4 @@ In [our gsearch fork](https://github.com/discoverygarden/gsearch) as of [version
 |Parameter|Default|Description|
 |---|---|---|
 |`index_ancestors`|`false`|Boolean flag: `true` to produce an `ancestors_ms` field (as used by the [islandora_collection_search](https://github.com/discoverygarden/islandora_collection_search) module); otherwise, `false` avoid generating. Also, note: When migrating objects between collections, it would be necessary to update all descendents to ensure their list of ancestors reflect the current state... We do this in the islandora_collection_search module when migrating, instead of reindexing all the descendents whenever indexing an object (updating a collection label would be fairly expensive if we blindly reindexed).|
+|`index_parent_models`|`false`|Boolean flag: `false` to produce an `parent_models_ms` field (as used by the [islandora_child_filter](https://github.com/discoverygarden/islandora_child_filter) module; otherwise, `false` avoid generating.|
```

Comment:
Description is messed up: `false` does the thing, but also prevents it from being done?